projects/1.19/assets/structurize/structurize/lang/zh_cn.json

```
@@ -0,0 +1,184 @@
+  "com.ldtteam.structurize.gui.scantool.chests": "（容器内容可能不会被列出）",
+  "com.ldtteam.structurize.gui.scantool.copy.ok": "\"%s\"扫描指令已复制。",
+  "com.ldtteam.structurize.gui.scantool.copy.notscan": "只能复制扫描指令。 点击以粘贴激活的扫描。",
+  "com.ldtteam.structurize.gui.scantool.paste.badcommand": "命令方块包含扫描指令之外的内容。 如果你想粘贴请先清除它。",
+  "com.ldtteam.structurize.gui.scantool.paste.badname": "你必须要为你的扫描输入一个文件名 (不包含空格).",
+  "com.ldtteam.structurize.gui.scantool.paste.different": "按住 CTRL 以确认将\"%s\"粘贴到\"%s\"的扫描器中。",
+  "com.ldtteam.structurize.gui.scantool.paste.ok": "粘贴当前的扫描 \"%s\".",
+  "com.ldtteam.structurize.gui.scantool.teleport.dimension": "你必须与你的建筑处于相同的维度。",
+  "com.ldtteam.structurize.gui.scantool.teleport.nocmd": "你必须至少复制或粘贴一次命令块来设置传送位置。",
+  "com.ldtteam.structurize.gui.scantool.teleport.noscan": "你必须在当前位置中设置扫描区域才能设置传送位置。",
+  "com.ldtteam.structurize.gui.shapetool.hollow": "中空",
+  "com.ldtteam.structurize.gui.shapetool.ignore": "忽略方块",
+  "com.ldtteam.structurize.gui.shapetool.pickfillblock": "选择填充方块",
+  "com.ldtteam.structurize.gui.shapetool.pickmainblock": "选择主要方块",
+  "com.ldtteam.structurize.gui.shapetool.replace": "替换方块",
+  "com.ldtteam.structurize.gui.shapetool.solid": "固体",
+  "com.ldtteam.structurize.gui.structure.edit.title": "结构路径：",
+  "com.ldtteam.structurize.gui.tagtool.addtag": "已添加标签%s到%s",
+  "com.ldtteam.structurize.gui.tagtool.anchor.notvalid": "这不是有效的锚点方块，只有能够存储 nbt 数据的方块实体才行（比如占位符方块）。",
+  "com.ldtteam.structurize.gui.tagtool.anchorsaved": "锚点方块实体已保存，现有标签列表已加载完毕。你现在可以对方块应用标签了！",
+  "com.ldtteam.structurize.gui.tagtool.apply": "应用标签",
+  "com.ldtteam.structurize.gui.tagtool.close": "关闭 GUI",
+  "com.ldtteam.structurize.gui.tagtool.currenttag": "可用标签：",
+  "com.ldtteam.structurize.gui.tagtool.discard": "放弃更改",
+  "com.ldtteam.structurize.gui.tagtool.discard.success": "重置标签为锚点标签",
+  "com.ldtteam.structurize.gui.tagtool.invalidread": "试图从无效的锚点方块实体中读取 tagPos 数据",
+  "com.ldtteam.structurize.gui.tagtool.invalidsave": "试图将 tagPos 数据保存到无效的锚点方块实体中",
+  "com.ldtteam.structurize.gui.tagtool.noanchor": "请先通过按住 SHIFT 并右击选择一个有效的锚点方块。",
+  "com.ldtteam.structurize.gui.tagtool.notag": "错误：先输入一个有效的标签再应用！",
+  "com.ldtteam.structurize.gui.tagtool.removed": "标签 %s 已从 %s 移除。",
+  "com.ldtteam.structurize.gui.tagtool.save": "成功使锚点方块应用了标签",
+  "com.ldtteam.structurize.network.messages.schematicsavemassage.toobig": "结构文件太大，不能大于 %s 字节！",
+  "com.structurize.command.playernotfound": "无法找到用以保存扫描结果的玩家！",
+  "com.structurize.command.scan.no.perm": "你无权用命令来进行扫描，请使用扫描工具！",
+  "com.structurize.command.scan.success": "扫描保存成功",
+
+  "com.structurize.command.paste.no.perm": "你无权用命令来进行扫描，请使用扫描工具！",
+  "com.structurize.command.paste.no.pack": "指定的包不存在",
+  "com.structurize.command.paste.no.blueprint": "指定的蓝图不存在",
+  "com.structurize.command.paste.success": "已成功触发粘贴",
+
+  "com.structurize.gui.buildtool.leave.tip": "用结构建筑工具右击某个固体方块来调整建筑位置",
+  "com.structurize.gui.buildtool.tip": "按 ESC 键关闭 GUI 以在世界中预览",
+  "key.structurize.categories.general": "结构化",
```

| **审查级别**    |
|--------------|
| ⚪ 轻微 / 不阻止合并但需三思  |

原文：**Structurize**

我也不知道哪里来的约定俗成

```suggestion
  "key.structurize.categories.general": "结构化（Structurize）",
```